### PR TITLE
Made a dropdown which allows you to sort ads by a price,date #125

### PR DESCRIPTION
**Pull request by BrandonBalala**

### CODE REVIEW

**Comment by kevcamel** [comment]
@BrandonBalala, while i was working on the Unit-Test branch I noticed you had implemented this feature in that branch as well. I did some CSS fixes to the alignment of the button in that branch. we possibly don't need this branch anymore?

File: app/Http/Controllers/ProductCategoryController.php [hunk 1]
```diff
@@ -13,32 +13,30 @@ public function show(ProductCategory $category, Request $request)
...
+        $sortBy_column = 'views';
+        $orderBy = 'desc';
+        break;
+        case 'newestFirst':
+        default:
+        $sortBy_column = 'created_at';
+        $orderBy = 'desc';
+        break;
+    }
+
```
Comment (laurie-y) [inline app/Http/Controllers/ProductCategoryController.php:39]: I think it's better to have the indents(indents on the breaks are nice too). It's easier to look at and more clearly defines the cases.

Comment (BrandonBalala) [reply app/Http/Controllers/ProductCategoryController.php:39]: Will do that future commits, I usually hit the reindent command on my IDE and that's what it did, so I left it as is.

Comment (laurie-y) [reply app/Http/Controllers/ProductCategoryController.php:39]: Sounds good 👍 Everything else looks good otherwise.

**Pull request merged**